feature/experimental/bgp/ate_tests/base_bgp_session_parameters/README.md

```diff
@@ -0,0 +1,44 @@
+RT-1.1: Base BGP Session Parameters
+
+Summary
+
+    Base BGP session establishment, with required capabilities
+
+Procedure
+
+    Establish BGP session on ATE and DUT with table-based configuration to cover:
+        ○ Explicitly specified Router ID
+        ○ MD5 password
+        ○ Explicitly specified local and remote AS number
+        ○ Explicitly specified local IP address
+        ○ Explicit holdtime and keepalive interval
+        ○ Explicitly specified retry interval
+    ● Validate session state and capabilities received on ATE.
+    ● Validate session state and capabilities received on DUT using telemetry.
+    ● Terminate BGP session using NOTIFICATION message and ensure that device telemetry correctly   reports the error codes.
```

Comment:
Instead of a separate TODO below, just mark the TODO here like this:

* TODO: Terminate BGP session ...
  * Sending notification from ATE is not supported in the Ondatra framework. Waiting for Ixia support.